.env

```diff
@@ -0,0 +1,18 @@
+DB_USER=user_postgres
+DB_PASSWORD=password_postgres
+DB_NAME=project_db
+DB_HOST=postgres_trade
+DB_PORT=5432
+NL_USERNAME=your_user_name
+NL_PASSWORD=your_password
+MAIL_ADMIN=your_mail
+MAIL_SERVER=your_mail
+MAIL_PORT=587
+MAIL_USE_TLS=True
+MAIL_USERNAME=your_mail_user
+MAIL_PASSWORD=your_password
+REDIS_HOST=redis_trade
+RABBITMQ_HOST=rabbitmq_trade
+RABBITMQ_USER=myuser1
+RABBITMQ_PASSWORD=mypass1
+RABBITMQ_VHOST=myvhost1
```

Comment:
You do not have a newline (usually '\n', aka CR or CRLF) at the end of file. It is a good style to always put the newline as a last character if it is allowed by the file format. Check editor settings.